### PR TITLE
For 1.20, drop requirement to have a Service associated with a pod

### PR DESCRIPTION
**Pull request by howardjohn**
Fixed in https://github.com/istio/istio/pull/46295

### CODE REVIEW

**Review by dhawton — APPROVED** [review]

**Comment by ericvn** [comment]
Since you mention 1.20, we will need to wait to merge this until after we branch at the 1.19.0 release.

**Review by ericvn — CHANGES_REQUESTED** [review]
Currently the master branch for istio.io is for the 1.19 release (until we branch it on release day. Hold this PR until then.

**Review by ericvn — APPROVED** [review]

**Comment by dhawton** [comment]
Removed DNM as master is 1.20

**Pull request merged**
